### PR TITLE
[DomCrawler] mark some methods as internal

### LINKED ISSUE #18465: [DomCrawler] Unusable containsOption method in ChoiceFormField class
The containsOption method of the Symfony\Component\DomCrawler\Field\ChoiceFormField class should check whether given value is in the existing options. But actually it doesn't check the existing options ($this->options) but expects the $options array as the second parameter. It is unusable for the form field testing as there's no way to get the private property $options of the ChoiceFormField class. I suppose that it should check the $options property instead of expecting it as an argument.

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DomCrawler/Field/ChoiceFormField.php#L281

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #18465
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/DomCrawler/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * Marked the `containsOption()`, `availableOptionValues()`, and `disableValidation()` methods of the
```
Comment (nicolas-grekas) [inline src/Symfony/Component/DomCrawler/CHANGELOG.md:7]: maybe worth an UPGRADE* entry?

Comment (xabbuh) [reply src/Symfony/Component/DomCrawler/CHANGELOG.md:7]: not sure, it seems we didn't do that before when marking methods or classes as final

**Review by derrabus — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/DomCrawler/CHANGELOG.md (hunk 1 above)
Comment (chalasr) [reply src/Symfony/Component/DomCrawler/CHANGELOG.md:7]: we did https://github.com/symfony/symfony/search?l=Markdown&q=final :)

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Component/DomCrawler/CHANGELOG.md (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Component/DomCrawler/CHANGELOG.md:7]: @chalasr I only see matches in the changelog files :)

**Comment by derrabus** [comment]
Thank you Christian.

**Pull request merged**